pluginDocumentation_customLocale.md

```diff
@@ -43,18 +43,18 @@ Templates
 		localeFile.tpl
 
 Database access, server access
------------------------------
-- reading access to OMP tables: 0
-- writing access to OMP tables: 0
+------------------------------
+- reading access to OJS/OMP/OPS tables: 0
+- writing access to OJS/OMP/OPS tables: 0
 - new tables: 0
 - nonrecurring server access: no
 - recurring server access: yes
 
 		creating, reading from and writing to files in [files_dir]/[presses|journals]/%contextId%/customLocale
  
 Classes, plugins, external software
------------------------
-- OMP classes used (php): 19
+-----------------------------------
+- OJS/OMP/OPS classes used (php): 19
```

Comment:
I think this kind of documentation isn't useful and is probably long out of date; rather than updating it, I'd recommend removing it.